syntheseus/search/mol_inventory.py

```diff
@@ -36,24 +39,41 @@ class ExplicitMolInventory(BaseMolInventory):
     """
 
     @abc.abstractmethod
-    def purchasable_mols(self) -> Collection[Molecule]:
-        """Return a collection of all purchasable molecules."""
+    def to_purchasable_mols(self) -> Collection[Molecule]:
```

Comment:
To avoid breaking backward compatibility, should we keep the `purchasable_mols` method but add a deprecation warning? It could just be an alias for `to_purchasable_mols`?